regolith/helpers/u_institutionshelper.py

```diff
@@ -0,0 +1,188 @@
+"""
+Helper for updating/adding  to the projecta collection.
+"""
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import all_docs_from_collection, fragment_retrieval
+from itertools import chain
+import uuid
+import datetime as dt
+
+TARGET_COLL = "institutions"
+
+def subparser(subpi):
+    subpi.add_argument("institution_id",
+                       help="id of the institution, e.g., columbiau.")
+    subpi.add_argument("-n","--name",
+                        help="Full name of the institution")
+    subpi.add_argument("-i","--index",
+                       help="Index of the item in the enumerated list to update.",
+                       type=int)
+    subpi.add_argument("--city",
+                       help="The city where the institution is. "
+                            "Required for a new institution.")
+    subpi.add_argument("--country",
+                       help="The country where the institution is. "
+                            "Required for a new institution")
+    subpi.add_argument("-a", "--aka",
+                       nargs='+',
+                       help="List of all the different names this "
+                            "institution is known by.")
+    subpi.add_argument("--state",
+                       help="The state where the institution is.")
```

Comment:
state and zip are required if the country is US btw.  Please could you add words to the help and code functionality to ensure this?  Not required if the institution is abroad